### ISSUE TITLE
[Blog post]: Nomad - K-ray Guide - How to use Local in low internet scenarios/flaky connections/5G

### ISSUE THREAD

**Issue by 6za** [kubefirst/kubefirst#832]
# Goal 

Create a guide to help nomad users trying to use `kubefirst installer` during: 
- Working from Hotel/Cellular based connections 
- Limited/Throttled networks   
- Conference floors.

The main focus for now is to have a `local` install covered.

**Comment by 6za** [comment]
Lessons learned on trying to use the installer with cellular/hotel connection. 

Some issues like this will happen when pulling images:
```bash 
Error response from daemon: Get "https://registry-1.docker.io/v2/": net/http: TLS handshake timeout
Error response from daemon: Head "https://registry-1.docker.io/v2/rancher/klipper-lb/manifests/v0.3.5": Get "https://auth.docker.io/token?account=----&scope=repository%3Arancher%2Fklipper-lb%3Apull&service=registry.docker.io": net/http: request canceled (Client.Timeout exceeded while awaiting headers) 
```

The possible cause, seems to be bad networking routes/timeout of the network provider in use that seems to be doing some bad job on managing routes. Either way, to address that the use of  VPN helps to get a better routing and isolating your traffic from the Hotel/Celular network provider.

**Comment by 6za** [comment]
Another lesson: 
Pre-load your docker registry, local will install several services on your k3d cluster, due to the way docker works, there is internal logic on docker image pulling mechanism to mitigate re-download artifacts already known locally. 


So, to prevent installation issue due to time-outs due to delays to get an image, you can do the following:

```bash 
cat <<EOF > ~/k1_images.txt
rancher/local-path-provisioner:v0.0.21
rancher/mirrored-coredns-coredns:1.9.1
rancher/klipper-helm:v0.7.3-build20220613
rancher/klipper-lb:v0.3.5
rancher/mirrored-metrics-server:v0.5.2
rancher/klipper-lb:v0.3.5
rancher/mirrored-library-traefik:2.6.2
public.ecr.aws/docker/library/redis:7.0.4-alpine
quay.io/argoproj/argocd:v2.4.8
ghcr.io/dexidp/dex:v2.30.2
quay.io/argoproj/argocd:v2.4.8
quay.io/jetstack/cert-manager-webhook:v1.9.1
quay.io/jetstack/cert-manager-cainjector:v1.9.1
quay.io/minio/minio:RELEASE.2022-09-17T00-09-45Z
hashicorp/vault-k8s:1.0.0
quay.io/jetstack/cert-manager-controller:v1.9.1
quay.io/argoproj/workflow-controller:v3.4.1
quay.io/argoproj/argocli:v3.4.1
ghcr.io/external-secrets/external-secrets:v0.5.6
ghcr.io/helm/chartmuseum:v0.15.0
public.ecr.aws/kubefirst/console:2.0.0
ghcr.io/runatlantis/atlantis:v0.19.2
hashicorp/vault:1.11.3
summerwind/actions-runner-controller:v0.25.2
quay.io/brancz/kube-rbac-proxy:v0.13.0
ghcr.io/external-secrets/external-secrets:v0.5.6
EOF
```

2 - pull all images before running local install

```bash 
cat  ~/k1_images.txt |   xargs -I {}   docker pull  {}
```

**Comment by 6za** [comment]
Another lesson - DNS

Other issue observed was, DNS name resolution of new IPs from cloud providers like AWS/Digital Ocean make take much longer to have a route available post the machine is created and network ready from the cloud provider. Giving the impression that the installation didn't worked, but it is just bad routing from the network/telecom provider.

**Comment by fharper** [comment]
@6za : do you want me to take all these comments, and write something out of it, or you'll do a PR to the docs yourself?

**Comment by 6za** [comment]
I am just waiting we get 1.11 to main, to be able to focus on solutions that apply to a code one main, not in a branch. :)

**Comment by fharper** [comment]
Perfect, let me know how I can help!

**Comment by fharper** [comment]
I will use the minimum information about Kubefirst to add a line or two in the documentation to fix #952 , and we can link to your blog post once it's published.
